05/app.js

```diff
@@ -1,53 +1,90 @@
 const stats = {
-    paragraphs: {
-        'p1': 0,
-    },
-    links: {
-        '/dolor.html': 0,
-    }
-};
+	paragraphs: {
+		p1: 0,
+	},
+	links: {
+		'/dolor.html': 0,
+	},
+}
 
 /* tutaj umieść swój kod */
+const pList = document.querySelectorAll('p')
 
+pList.forEach(function (item) {
+	item.addEventListener('click', function (e) {
+		e.preventDefault()
 
-/* nie modyfikuj kodu poniżej, ale przeanalizuj go */
+		if (e.target.getAttribute('href') === null) {
+			countParagraphs(e)
+		} else if (e.target.getAttribute('href') !== null) {
```

Comment:
Jeśli szlibyśmy w tą stronę to wystarczy `else` bo jeśli coś ma `null` to w odwrotnym przypadku wiadomo, że nie ma `null` i nie trzeba tego sprawdzać przez dodatkowy warunek.

Ja natomiast w tym ifie sprawdzałbym czy coś jest tagiem `P` lub `A` :)